.github/workflows/get-published-images-scan-and-report.yaml

```diff
@@ -0,0 +1,37 @@
+name: Get ROCKs modified and build-scan-test-publish them
+
+on:
+  workflow_call:
+    secrets:
+      GH_TOKEN:
+        required: true
+    inputs:
+      report-vulnerabilities:
+        description: "Whether to report security vulnerabilities through Github issues."
+        required: false
+        default: false
+        type: boolean
+      severity:
+        description: "Comma separated list of severities of vulnerabilities to scanned for and displayed"
+        required: false
+        type: string
+        default: "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL"
+
+
+jobs:
+  get-published-images-names:
+    name: Get published images names
+    uses: ./.github/workflows/get-published-image-names.yaml
+
+  scan-report-vulnerability:
```

Comment:
What about a concise name for github UI like
```suggestion
  scan-report-vulnerability:
    name: Scan and report
```

Reply:
I would like us to avoid [this](https://github.com/canonical/charmed-kubeflow-workflows/issues/49) happening again